tooling/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "private": true,
   "scripts": {
-    "lint": "jscs"
+    "lint": "cd .. && jscs"
```

Comment:
`pushd .. && jscs && popd`

Lets use ^ so we do not mutate directory state... thus making lint re-entrant